plugins/fclrave/fclcollision.h

```diff
@@ -250,6 +247,9 @@ class FCLCollisionChecker : public OpenRAVE::CollisionCheckerBase
 
     void _PrintCollisionManagerInstanceLE(const KinBody::Link& link, FCLCollisionManagerInstance& envManager);
 
+    /// \brief get information from fcl space GetEnvBodies.
+    bool _GetEnvBodiesInfoFromFCLSpace(ostream& sout, istream& sinput);
```

Comment:
@snozawa is this for debug purposes? Or are you going to be using it in production code? If so, where?

Reply:
thanks for comment!

This is only for debugging purpose to write more explicit unit test codes to observe the internal states of `fclcollision/fclspace` on various events, such as loading json, adding `KinBody`, removing `KinBody`, cloning stuff, ..etc.